DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -64,7 +64,7 @@ public class PortalSecurity
             new Regex("</form[^><]*>", RxOptions),
             new Regex("onerror", RxOptions),
             new Regex("onload", RxOptions),
-            new Regex("onfocus", RxOptions),
+            new Regex("\bonfocus\b", RxOptions),
```

Comment:
It would seem like we'd want this change for all of the `onwhatever` expressions, right?